src/gcp_scanner/crawl.py

```diff
@@ -627,8 +623,8 @@ def get_cloudfunctions(project_id: str,
         parent=f"projects/{project_id}/locations/-")
     while request is not None:
       response = request.execute()
-      for function in response.get("functions", []):
-        functions_list.append(function)
+      functions_list = [function 
+                        for function in response.get("functions", [])]
```

Comment:
This is redundant. Replace with `functions_list = response.get("functions", [])`

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.